.travis.yml

```diff
@@ -11,62 +11,72 @@ matrix:
      python: 3.7
    - env: TOXENV=py37-dj32-wt42
      python: 3.7
-   - env: TOXENV=py37-dj40-wt41
-     python: 3.7
-   - env: TOXENV=py37-dj40-wt42
+   - env: TOXENV=py37-dj32-wt50
      python: 3.7
    - env: TOXENV=py37-dj41-wt41
      python: 3.7
    - env: TOXENV=py37-dj41-wt42
      python: 3.7
+   - env: TOXENV=py37-dj41-wt50
+     python: 3.7
+   - env: TOXENV=py37-dj42-wt50
```

Comment:
I checked these and saw that this might be a good test matrix config:
```
py{38,39,310}-dj{40}-wt{41,42}
py{38,39,310}-dj{32,41}-wt{41,42,50}
py{311}-dj{41}-wt{41,42,50}
py{311}-dj{42}-wt{50}
```

and so I updated the yml file accordingly